os/arch/Kconfig.chip

```diff
@@ -13,6 +13,7 @@ config ARCH_ARM
 	select ARCH_HAVE_VFORK
 	select ARCH_HAVE_STACKCHECK
 	select ARCH_HAVE_CUSTOMOPT
+	select ARCH_HAVE_SETJMP
```

Comment:
This change is a different kind of change. Let's split commit.

Reply:
Okay, will do.